AUTHORS.rst

```diff
@@ -26,3 +26,4 @@ considered "The PyVista Developers":
 - Matthew Flamm, (`@MatthewFlamm <https://github.com/MatthewFlamm>`_)
 - Darik Gamble, (`@darikg <https://github.com/darikg>`_)
 - Bram De Cooman, (`@dcbr <https://github.com/dcbr>`_)
+- `@user27182 <https://github.com/user27182>`_
```

Comment:
@pyvista/maintainers  I have determined that this change is copyrighted and have added @user27182 to `AUTHORS.rst`.
@user27182 This change is my suggestion, so please make it as you wish.

Reply:
@tkoyama010 what do you mean by "copyrighted"?

Reply:
@adeak Thanks for comment. Meaning they is into this category :)
https://github.com/pyvista/pyvista/blob/75febafe79be9482accc2af931f1978fd4bec7ee/AUTHORS.rst?plain=1#L10-L12

Reply:
I agree, and I'd like to further emphasize that the change here isn't copyrighted, which might imply that there are license issues.  It is that the cumulative contributions that rise to the level of being added as a mentioned "PyVista Developer".  I'm also +1 on the suggestion if accepted.

Reply:
Thanks! I enjoy contributing to PyVista because of the help and support from the community (and the CI testing is fantastic). I'll leave the changes to `AUTHORS.rst` as-is for now.